src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -243,9 +243,9 @@ private IReadOnlyCollection<WindowScanOutput> ScanSyncWithProvisionForBuildAgent
             {
                 return scanner.Scan(null).WindowScanOutputs;
             }
-            catch (AxeWindowsAutomationException e)
+            catch (AxeWindowsAutomationException)
```

Comment:
Maybe this should be all Exception types?
```suggestion
            catch (Exception)
```